### PR TITLE
Remove job .tsx files from preferences menu, use compiled data instead

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Closes #63172, since it's included.

Removes every job .tsx file, and moves their necessary data over to the job datums themselves. This lowers the burden of both adding and removing jobs, and allows for jobs to be removed without touching any internal code (I'm specifically doing this for events)

### CODE REVIEW

**Comment by Mothblocks** [comment]
Dubs

File: code/modules/jobs/job_types/prisoner.dm [hunk 1]
```diff
@@ -13,6 +14,10 @@
 	plasmaman_outfit = /datum/outfit/plasmaman/prisoner
 
 	display_order = JOB_DISPLAY_ORDER_PRISONER
+	department_for_prefs = /datum/job_department/security
+	departments_list = list(
+		/datum/job_department/service,
```
Comment (JohnFulpWillard) [inline code/modules/jobs/job_types/prisoner.dm:19]: HOLY SHIT IT'S BACK BABY IT'S FUCKING BACK HOLY SHIT

Comment (Mothblocks) [reply code/modules/jobs/job_types/prisoner.dm:19]: Wait I didn't even mean to have both of these

File: code/modules/client/preferences/README.md [hunk 2]
```diff
@@ -363,30 +363,6 @@ Middleware can inject its own data at several points, such as providing new UI a
 
 ---
 
-## Jobs
-
-Every job must have an associated `.ts` file so that the UI knows where to place it.
-
-Create a file in `tgui/packages/tgui/interfaces/PreferencesMenu/jobs/jobs/`.
-
-This will specify the description and department of the job. Here is the details for the medical doctor:
```
Comment (Mothblocks) [inline code/modules/client/preferences/README.md:372]: Never even caught that this was the cook and not the medical doctor

I just didnt want to show the backslash thing

**Comment by optimumtact** [comment]
even easier to bikeshed

**Review by tralezab — APPROVED** [review]
dm side looked spotless, you probably want a tsx wizard like style or action to take a peek tho

File: code/modules/jobs/job_types/prisoner.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/jobs/job_types/prisoner.dm]: NOOOOOOOOOOOOOOOO

**Pull request merged**